app/views/applications.njk

```diff
@@ -38,22 +38,63 @@
       </div>
     {% else %}
       <div class="govuk-grid-row">
-        {{ filter(availableStatus, selectedStatus) }}
+        <div class="govuk-grid-column-one-quarter govuk-!-margin-top-5">
+          {%- from "moj/components/filter/macro.njk" import mojFilter -%}
+          {%- set filterOptionsHtml %}
+          {{ govukCheckboxes({
```

Comment:
Could this be a model which is part of the added to the view in the route? This would make the .njk file a lot neater and bloated.

Reply:
agree